CMakeLists.txt

```diff
@@ -57,6 +57,16 @@ set(THEROCK_ARTIFACT_ARCHIVE_SUFFIX "" CACHE STRING "Suffix to add to artifact a
 
 option(THEROCK_BUNDLE_SYSDEPS "Builds bundled system deps for portable builds into lib/rocm_sysdeps" ON)
 
+# Source settings.
+option(THEROCK_USE_EXTERNAL_ROCM_LIBRARIES "Use the `rocm-libraries` monorepo instead of submodules" OFF)
```

Comment:
Please add a link to https://github.com/ROCm/rocm-libraries, either in the `# comment` and/or help text.

Reply:
Sure, I added a comment.